myapp/.env

```diff
@@ -60,5 +60,5 @@ KEY=rHbaqmfdhmdrDDPIytYhwSRzcvpOesjZ
 SPOTIFY_ID=bcd5e565bc464208a1a03fde424733a4
 SPOTIFY_SECRET=1ec3def0425e4f85a2f3378b221c23bf
 
-# Localhost url
-LOCALHOST_URL=http://localhost:4000
+# Spotify Callback URL
+SPOTIFY_CALLBACK_URL=http://localhost:4000
```

Comment:
If this is going to be only for the callback URL we can set it to the entire URL instead of concatenating later

Reply:
✔️ will change that back, then